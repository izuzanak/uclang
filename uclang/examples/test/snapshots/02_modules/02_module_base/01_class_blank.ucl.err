 ---------------------------------------- 
Exception: ERROR: in file: "01_class_blank.ucl" on line: 56
      ("simple == blank -> %d\n" % (simple == blank)).print();

Class Simple does not contain method operator_binary_double_equal#1
 ---------------------------------------- 
