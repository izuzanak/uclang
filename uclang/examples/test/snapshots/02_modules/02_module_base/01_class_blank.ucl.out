
=== Blank object ===============================================================

Blank: <blank>
Blank: <blank>

=== Blank assign operator ======================================================

blank: <blank>
blank: 10

=== Blank equality operators ===================================================

blank == 10    -> 0
blank == blank -> 1
Blank == blank -> 1
blank != 10    -> 1
blank != blank -> 0
Blank != blank -> 0

=== Blank equality example =====================================================

blank == simple -> 0

=== Blank methods - compare#1 ==================================================

blank.compare(blank) -> 0
blank.compare(10)    -> -1

=== Blank methods - to_string#0 ================================================

blank.to_string() -> <blank>
Blank.to_string() -> <blank>

=== Blank methods - print#0 ====================================================

blank.print: <blank>

=== Tutorial end ===============================================================

